Kitodo/src/main/java/org/kitodo/production/services/ocr/OCRWorkflowService.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.services.ocr;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import org.kitodo.data.database.beans.OCRWorkflow;
+import org.kitodo.data.database.exceptions.DAOException;
+import org.kitodo.data.database.persistence.OCRWorkflowDAO;
+import org.kitodo.data.exceptions.DataException;
+import org.kitodo.production.services.ServiceManager;
+import org.kitodo.production.services.data.base.SearchDatabaseService;
+import org.primefaces.model.SortOrder;
+
+public class OCRWorkflowService extends SearchDatabaseService<OCRWorkflow, OCRWorkflowDAO> {
+
+    private static volatile OCRWorkflowService instance = null;
+
+    /**
+     * Constructor necessary to use searcher in child classes.
+     */
+    public OCRWorkflowService() {
```

Comment:
Should this method not be ` private` as a public class constructor ruins the ` getInstance()` usage?